analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -315,7 +315,7 @@ def handle_failure(source_analyzer, rh, zip_file, result_base, actions_map):
         zip_file,
         map(lambda path: os.path.join(action.directory, path), other_files))
 
-    with zipfile.ZipFile(zip_file, 'w') as archive:
```

Comment:
This was changed [recently](https://github.com/Ericsson/codechecker/commit/7d6719ac15ab732e6f82608bd91d18ff1099e063) to overwrite the zip files if a newer analysis is run.
I think in this case we should handle this part differently.